atef/bin/config.py

```diff
@@ -18,6 +19,44 @@ def build_arg_parser(argparser=None):
     if argparser is None:
         argparser = argparse.ArgumentParser()
 
+    # Arguments that need to be passed through to Qt
+    qt_args = {
+        '--qmljsdebugger': 1,
+        '--reverse': '?',
+        '--stylesheet': 1,
+        '--widgetcount': '?',
+        '--platform': 1,
+        '--platformpluginpath': 1,
+        '--platformtheme': 1,
+        '--plugin': 1,
+        '--qwindowgeometry': 1,
+        '--qwindowicon': 1,
+        '--qwindowtitle': 1,
+        '--session': 1,
+        '--display': 1,
+        '--geometry': 1
+    }
+
+    for name in qt_args:
+        argparser.add_argument(
+            name,
+            type=str,
+            nargs=qt_args[name]
+        )
+
+    argparser.add_argument(
+        '--style',
+        type=str,
+        choices=QStyleFactory.keys(),
+        default='fusion',
+        help='Qt style to use for the application'
+    )
+
+    argparser.description = """
+    Runs the atef configuration GUI, optionally with an existing configuration.
+    Qt arguments are also supported. For a full list, see the Qt docs: https://doc.qt.io/qt-5/qguiapplication.html#supported-command-line-options
+    """
```

Comment:
```suggestion
    Qt arguments are also supported. For a full list, see the Qt docs:
    https://doc.qt.io/qt-5/qapplication.html#QApplication
    https://doc.qt.io/qt-5/qguiapplication.html#supported-command-line-options
    """
```

Reply:
Oddly enough, the args here are split between those two different docs pages with no linkage between them...

Reply:
I think you're right, which is why it's perplexing that the docs are separated like this...

Reply:
I bet their documentation generator doesn't understand the difference between type aliases and the underlying type itself.